virt/backup_restore/virt-backup-restore-overview.adoc

```diff
@@ -1,23 +1,28 @@
 :_content-type: ASSEMBLY
 include::_attributes/common-attributes.adoc[]
 [id="virt-backup-restore-overview"]
-= Backup and restore overview
+= Backing up and restoring virtual machines
 :context: virt-backup-restore-overview
 
 toc::[]
 
-As a cluster administrator, you back up and restore {VirtProductName} virtual machines (VMs) by using the OpenShift API for Data Protection (OADP).
-// Non-admin user [https://issues.redhat.com/browse/OADP-203] is targeted for OADP 1.1.
+You back up and restore virtual machines by using the xref:../../backup_and_restore/index.adoc#application-backup-restore-operations-overview[OpenShift API for Data Protection (OADP)].
 
-You install the OADP Operator by using Operator Lifecycle Manager. The Operator installs link:https://velero.io/docs/v1.7/[Velero 1.7]. You create a `Secret` for the backup storage provider and then you install the Data Protection Application.
+.Prerequisites
 
-You back up VMs by creating a `Backup` custom resource (CR).
+* Access to the cluster as a user with the `cluster-admin` role.
+// Non-admin user [https://issues.redhat.com/browse/OADP-203] is targeted for OADP 1.2.
 
-You restore the `Backup` CR by creating a `Restore` CR.
+.Procedure
 
+. Install the xref:../../backup_and_restore/application_backup_and_restore/installing/about-installing-oadp.adoc#about-installing-oadp[OADP Operator] according to the instructions for your storage provider.
+. Install the xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc#oadp-installing-dpa_installing-oadp-ocs[Data Protection Application] with the `kubevirt` and `openshift` xref:../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[plugins].
+. Back up virtual machines by creating a xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#backing-up-applications[`Backup` custom resource (CR)].
+. Restore the `Backup` CR by creating a xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/restoring-applications.adoc#restoring-applications[`Restore` CR].
+
+[role="_additional-resources"]
 [id="additional-resources_{context}"]
```

Comment:
`s/{context}/virt-backup-restore-overview` per our guidelines. I think we use the variable {context} in modules.